.vib/postgresql/goss/vars.yaml

```diff
@@ -0,0 +1,26 @@
+binaries:
+  - gosu
+  - postgres
+  - psql
+  - pg_dump
+  - pg_dumpall
+  - pg_restore
+files:
+  - paths:
+    - /opt/bitnami/postgresql/conf/pg_hba.conf
+    - /opt/bitnami/postgresql/conf/postgresql.conf
```

Comment:
Checking `postgresql.conf` exists may not be enough. The function `postgresql_create_config()` performs additional steps after creating it, please check the file contains the added line.

Reply:
I've added an extra test checking if the conf file contains the include_dir parameter.

Reply:
Nice! If you want, you could remove the `postgresql.conf` path from here as you already have the tests in `postgresql.yaml`. Besides that, the PR LGTM.